Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,22 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array != null && array.Length != 0)
```

Comment:
Remember the techniques to de-nest?
```cs
if(array == null || array.Lenght == 0) return;
for (int i = 0; i <= array.Length - 1; i++)
{
  SingleFunctionCall();
}
```

Reply:
Much more readable like this as you don't have to keep track of all the things that happened/is happening above.